CHANGES.md

```diff
@@ -7,12 +7,14 @@
 ##### Additions :tada:
 
 - Vertical exaggeration can now be applied to a `Cesium3DTileset`. Exaggeration of `Terrain` and `Cesium3DTileset` can be controlled simultaneously via the new `Scene` properties `Scene.verticalExaggeration` and `Scene.verticalExaggerationRelativeHeight`. [#11655](https://github.com/CesiumGS/cesium/pull/11655)
+- The `Cesium3DTileset.dynamicScreenSpaceError` optimization is now enabled by default, as this improves performance for street-level horizon views. [#11718](https://github.com/CesiumGS/cesium/pull/11718)
```

Comment:
1. This will be a breaking change, so it should be documented in the breaking change section.
2. Let's move this to next month's release given today's date.

Reply:
I moved the entry. I also added a couple entries for the changed defaults (although one value is still pending on our discussion). If that's too much detail, I could combine the entries.